app/src/main/res/values-sw600dp-port/dimens.xml

```diff
@@ -392,6 +393,8 @@
   <dimen name="solution_summary_margin_bottom_linear_layout">32dp</dimen>
   <dimen name="solution_summary_margin_top_solution_summary_label">16dp</dimen>
 
+  <!-- Story Fragment dimens. -->
```

Comment:
<!-- Story Fragment dimens -->

Also define new values at the bottom

Reply:
Okk.